json-2017-02-12/README.md

```diff
@@ -0,0 +1,24 @@
+This benchmark finds an [assertion failure](https://bugs.chromium.org/p/oss-fuzz/issues/detail?id=641) in [json](https://github.com/nlohmann/json).
+
+As of August 31, 2017, it's usually found in about 5 minutes using the provided
+seed.
+
+```
+json-2017-02-12-libfuzzer: BUILD/test/src/fuzzer-parse_json.cpp:50: int LLVMFuzzerTestOneInput(const uint8_t *, size_t): Assertion `s1 == s2' failed.
+==3260== ERROR: libFuzzer: deadly signal
+    #0 0x4eafb3  (json-2017-02-12-libfuzzer+0x4eafb3)
```

Comment:
this is very weird. but given this is an assert, the stack trace is irrelevant, please remove everything but the first line.